### ISSUE TITLE
Initiate specific step on load

### ISSUE THREAD

**Issue by Anandks1993** [americanexpress/react-albus#56]
I want to start wizard from specific step. On mount step comes with id as null. Please help me to achieve this.

Eg: Consider wizard is having 3 steps and I want to start from 2nd step on mount, after that the flow should be normal.

**Comment by jsmartfo** [comment]
Does the skip step example illustrate what you need? http://americanexpress.io/react-albus/

**Comment by Anandks1993** [comment]
Yeah, but on mount I am getting step with id as null and steps with empty array. I want to skip the first step.

**Comment by micabe** [comment]
Hi @Anandks1993 ssee here: https://github.com/americanexpress/react-albus/blob/master/__tests__/components/Wizard.spec.jsx#L125

**Comment by herrmannplatz** [comment]
Stumbled across the same issue. My assumption was, that i can just pass `step: { id: 'theStepIWantToStartFrom'}` to the wizardprops
